auditbeat/Makefile

```diff
@@ -3,6 +3,7 @@ BEAT_TITLE=Auditbeat
 BEAT_DESCRIPTION=Audit the activities of users and processes on your system.
 SYSTEM_TESTS=false
 TEST_ENVIRONMENT=false
+GOX_OS?=linux darwin ## @Building List of all OS to be supported by "make crosscompile".
```

Comment:
@andrewkroh @adriansr Please advice if these are the correct platforms to crosscompile to.

Reply:
+ windows

Reply:
added